assets/locales/ja.json

```
@@ -27,7 +27,7 @@
   "2月17日~21日": "2月17日~21日",
   "2月25日~28日": "2月25日~28日",
   "人": "人",
-  "単位：": "単位：",
+  "単位: ": "単位: ",
```

再修正になってしまうのですが、翻訳キー・原文のほうにはスペースを含めず、Vue側で半角スペースを入れたほうが良いかもしれません（Transifexで翻訳者がスペースを入れ忘れる可能性が高いため）
```suggestion
  "単位:": "単位:",
```

---

了解です